intro/requirements.txt

```diff
@@ -0,0 +1,7 @@
+matplotlib==3.1.1
+numpy>=1.16.0
```

Comment:
might want to pin at 1.16.4, fewest error messages with tf. otherwise, don't need this, not importing np explicitly here